corehq/apps/app_manager/forms.py

```diff
@@ -76,30 +76,30 @@ def clean(self):
 
 class PromptUpdateSettingsForm(forms.Form):
     app_prompt = forms.ChoiceField(
-        label=ugettext_lazy("Prompt Updates to Latest Released App Version"),
+        label=ugettext_lazy("Prompt Updates to App"),
         choices=(
             ('off', ugettext_lazy('Off')),
             ('on', ugettext_lazy('On')),
             ('forced', ugettext_lazy('Forced')),
         ),
         help_text=ugettext_lazy(
             "If enabled, users will receive in-app prompts to update "
-            "to the latest released version of the app, if they are not "
+            "to update to the selected version of the app, if they are not "
             "already on it. (Selecting 'Forced' will make it so that users "
             "cannot continue to use CommCare until they update)"
         )
     )
 
     apk_prompt = forms.ChoiceField(
-        label=ugettext_lazy("Prompt Updates to Latest CommCare Version"),
+        label=ugettext_lazy("Prompt Updates to CommCare"),
         choices=(
             ('off', ugettext_lazy('Off')),
             ('on', ugettext_lazy('On')),
             ('forced', ugettext_lazy('Forced')),
         ),
         help_text=ugettext_lazy(
             "If enabled, users will receive in-app prompts to update "
-            "to the latest version of CommCare, if they are not already "
+            "to update to the selected version of CommCare, if they are not already "
```

Comment:
I think this now reads "to update to update to the selected..."

Reply:
Oops, thanks, fixed it.